volatility3/framework/plugins/windows/callbacks.py

```diff
@@ -2,16 +2,25 @@
 # which is available at https://www.volatilityfoundation.org/license/vsl-v1.0
 #
 
-import logging
 import contextlib
-from typing import List, Iterable, Tuple, Optional, Union
-
-from volatility3.framework import constants, exceptions, renderers, interfaces, symbols
+import logging
+from typing import Dict, Iterable, List, Optional, Tuple, Union, cast
+
+from volatility3.framework import (
+    constants,
+    exceptions,
+    interfaces,
+    objects,
+    renderers,
+    symbols,
+)
 from volatility3.framework.configuration import requirements
+from volatility3.framework.interfaces.objects import ObjectInterface
```

Comment:
Please try not to import classes directly, but only ever modules.  Importing `ObjectInterface` like this may allow someone to one day write `from callbacks import ObjectInterface` and it then won't be clear that it actually belongs to another module.